android/src/main/kotlin/com/getpinwheel/pinwheel/PinwheelPlugin.kt

```diff
@@ -178,10 +200,7 @@ internal class NativeView(context: Context, messenger: BinaryMessenger, id: Int,
       token = ""
     }
 
-    webView = WebView(context)
-    webView?.let {
-      Pinwheel.init(it, readLinkToken(), pinwheelEventListener)
-    }
+    pinwheelView = PinwheelViewGroupManager.init(context, readLinkToken(), pinwheelEventListener, "react native", "3.0.0")
```

Comment:
This needs to be changed to flutter, but I think flutter needs to be added to the allowlist. cc @roscioli

Reply:
roger can implement in link

Reply:
this is in production now for Link